src/content/operations/nat-traversal/_index.en.md

```diff
@@ -0,0 +1,415 @@
+to them then it's recommended to use a separate UDP port for every node marked
+as gateway in your cluster, this will allow eventual port mapping on a router if
+when communicating to clusters on remote sites with no direct routing.
+
+{{% notice note %}}
+If your cluster is behind a router which will NAT your traffic it's recommended to
+map the open ports into your gateway node private IPs, see the port mapping
+section. It could temporarily work without mapping, because most routers when
+performing NAT to the external network will not randomize or modify the source
+port of packets, but **this will happen as soon as two connections collide
+over the same source port**.
+{{% /notice %}}
+
+## UDP Dataplane protocol (IPSEC & wireguard)
+
+By default submariner uses the 4500 UDP port for the dataplane. The cluster-wide default
+can be changed with the `subctl join` `--nattport` flag.
+
+Although, it's more advisable to specify the port to be used per gateway node:
+
+```bash
+kubectl label node $NODE gateway.submariner.io/udp-port=4501
+```
+
+This allows individual gateways on the cluster to have different port numbers,
+hence allowing individual port-mapping if a public IP is shared.
+
+## IPsec
+
+### ESP or UDP Encapsulation
+
+IPsec in the libreswan cable driver will be configured for ESP protocol (more performant)
+whenever that is possible, that's normally when NAT is not detected, likely to happen
+when connectivity over the private IP is possible.
+
+If your network and routers filter the `IP>ESP` packets, encapsulation can be forced
+by using the `--force-udp-encaps` during subctl join, or by setting the operator `
+
+## Practical examples
+
+### All private and routed
+
+This is the simplest practical case, in this case all gateways can contact
+all other gateways via routing on their private IPs, and no NAT happens.
+
+<!-- source: https://app.diagrams.net/#G1IORJ9qW95qJ15P6Xk6idEfI4jj9Uo84l -->
+![All private and routed](/images/natt/nat_scenario_all_private_routed.svg)
+
+Two configuration alternatives exist:
+
+* Enabling the NATT discovery protocol on all gateway nodes:
+
+```bash
+kubectl label node $GWA --kubeconfig A gateway.submariner.io/natt-discovery-port=4490
+kubectl label node $GWB --kubeconfig B gateway.submariner.io/natt-discovery-port=4490
+kubectl label node $GWC --kubeconfig C gateway.submariner.io/natt-discovery-port=4490
+kubectl label node $GWD --kubeconfig D gateway.submariner.io/natt-discovery-port=4490
+
+# restart the gateways to pick up the new setting
+for cluster in A B C D;
+do
+  kubectl delete pod -n submariner-operator -l app=submariner-gateway --kubeconfig $cluster
+done
+```
+
+* Deploy the clusters with `subctl join --natt=false`
+
+{{% notice note %}}
+The first alternative is the preferred one, since it will allow the eventual
+addition of a cluster on a public cloud, or a remote site with no direct IP
+reachability or NATed reachability.
+{{% /notice %}}
```

Comment:
I assume we don't need this section since NATT discovery is enabled by default.

Reply:
Same comments as above, I will do the default-enabled update on a second PR, to backport this update first into website 0.9

Reply:
Done.